package.json

```diff
@@ -1,21 +1,38 @@
 {
   "name": "rudderstack-shopify-tracker",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Why are we bumping the version? The functionality is unchanged.

Reply:
addressed